governance/veANGLE/voting.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Details about the voting power of veANGLE holders
+---
+
+# 🗳 veANGLE Voting Power
+
+veANGLE owners hold the main voting power over the Angle Protocol. With their veANGLE balance, they can vote on protocol improvement proposals and gauge weights distribution.
+
+## Voting on improvement proposals
+
+veANGLE holders will be the main voting power of the Angle protocol. As such, they will be shaping the future of the protocol.
+
+To facilitate the voting process, the governance module of the protocol will change after the upgrade. Votes will still happen on Snapshot, and will then get executed by a multi-sig. This multi-sig will be composed of a mix of core team members and external people, including public figures if possible (anon or not) to increase accountability.
+
+## Voting on gauge weights
+
+veANGLE holders are also the ones voting on how the weekly ANGLE rewards are distributed between the different staking contracts (called gauges after the upgrade).
+
+This vote will be held weekly following the current mining liquidity program.
```

Comment:
Vote will not be held weekly, like Snapshot on the vote is going to be taken weekly but people will only be able to update their gauge votes every two weeks